clients/fakes/bmp_client_fake.go

```diff
@@ -91,6 +91,6 @@ func (bc *FakeBmpClient) Login(username string, password string) (clients.LoginR
 	return bc.LoginResponse, bc.LoginErr
 }
 
-func (bc *FakeBmpClient) CreateBaremetal(createBaremetalInfo clients.CreateBaremetalInfo, dryRun bool) (clients.CreateBaremetalResponse, error) {
-	return bc.CreateBaremetalResponse, bc.CreateBaremetalErr
+func (bc *FakeBmpClient) CreateBaremetals(createBaremetalsInfo clients.CreateBaremetalsInfo, dryRun bool) (clients.CreateBaremetalsResponse, error) {
```

Comment:
I also typically keep track of the parameters passed to a fake method implementation in a public variable